common/app/views/main.scala.html

```diff
@@ -28,12 +28,12 @@
 }
 
 @immersiveMainMedia(page: GalleryPage) = {
-    <div class="new-gallery content tonal--tone-media immersive-main-media">
+    <div class="new-gallery content tonal--tone-media immersive-main-media js-background-image">
```

Comment:
i wonder if `js-background-image` is too generic? maybe `js-objectfit-pf` or something?